src/main/kotlin/nl/info/zac/notification/Channel.kt

```diff
@@ -21,7 +21,8 @@ enum class Channel(private val code: String, val resourceType: Resource) {
     INFORMATIEOBJECTTYPEN("informatieobjecttypen", Resource.INFORMATIEOBJECTTYPE),
     OBJECTEN("objecten", Resource.OBJECT),
     ZAKEN("zaken", Resource.ZAAK),
-    ZAAKTYPEN("zaaktypen", Resource.ZAAKTYPE);
+    ZAAKTYPEN("zaaktypen", Resource.ZAAKTYPE),
+    CALLBACK_URL_TEST("test", Resource.CALLBACK_URL_TEST);
```

Comment:
We should keep the name in line with the other names. "zaken" channel is ZAKEN, "zaaktypen" is ZAAKTYPEN, so "test" should be TEST:

```suggestion
    TEST("test", Resource.CALLBACK_URL_TEST);
```